django/db/migrations/operations/fields.py

```diff
@@ -62,14 +62,18 @@ def deconstruct(self):
             kwargs
         )
 
-    def state_forwards(self, app_label, state):
+    @property
+    def actual_field(self):
```

Comment:
I don't like the name `actual_field`. More something like `intermediate_field` or `migrated_field`?